src/satellite/registry.ts

```diff
@@ -164,17 +164,14 @@ export class GlobalRegistry extends BaseRegistry {
       throw Error(`invalid config: ${foundErrors}`);
     }
 
-    // FIXME: what should these be?
-    const defaultAddress = `${config.app}-${config.env}.electric-sql.com`
-    const defaultPort = 5133
-
     const satelliteClientOpts = {
       ...satelliteClientDefaults,
       app: config.app,
-      env: config.env,
+      environment: config.env,
```

Comment:
This key is wrong, it typechecks because it's a superset of a required type, since `env` key is present in the `satelliteClientDefaults`

![image](https://user-images.githubusercontent.com/1357760/207557809-d751f302-1515-4c88-a62f-76d748c3ce8e.png)

Reply:
cool. Strengthening the type... some of this code with Opts types has a lot of unnecessary redundancies and don't make enough use of types. Getting better. Thanks.